easybuild/easyconfigs/r/reframe-cscs-tests/reframe-cscs-tests-22.07.eb

```diff
@@ -0,0 +1,41 @@
+# @author: manitart
+
+easyblock = 'Tarball'
+
+name = 'reframe-cscs-tests'
+version = '22.07'
+
+homepage = 'https://github.com/eth-cscs/cscs-reframe-tests'
+description = 'ReFrame test suite of CSCS.'
+
+toolchain = SYSTEM
+
+dependencies = [('reframe', '3.11.1')]
```

Comment:
```suggestion
dependencies = [('reframe', '3.12.0')]
```

Reply:
Is it worth updating the dependency to the latest `reframe` module available in production?